docs/resources/Guild.md

```diff
@@ -594,11 +602,11 @@ Modify the behavior and settings of an [integration](#DOCS_RESOURCES_GUILD/integ
 
 ###### JSON Params
 
-| Field               | Type    | Description                                                                                                                                 |
-| ------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
-| expire_behavior     | integer | the behavior when an integration subscription lapses (see the [integration](#DOCS_RESOURCES_GUILD/integration-object) object documentation) |
-| expire_grace_period | integer | period (in seconds) where the integration will ignore lapsed subscriptions                                                                  |
-| enable_emoticons    | boolean | whether emoticons should be synced for this integration (twitch only currently)                                                             |
+| Field               | Type    | Description                                                                                                                                                     |
+| ------------------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| expire_behavior     | integer | the behavior when an integration subscription lapses (see the [integration expire behaviors](#DOCS_RESOURCES_GUILD/integration-expire-Behaviors) documentation) |
```

Comment:
this link does not look correct.. it should be `/integration-object-integration-expire-behaviors`